addon/middleware/index.js

```diff
@@ -0,0 +1,13 @@
+//only here as a vanilla pass thru so we can
```

Comment:
Actually this file is an artifact of https://github.com/ember-redux/ember-redux/pull/121#discussion_r114086961, right?

Also, depending on the resolution of that comment, could this file be simply an `export default [];` or does that not work/am I missing something else?